src/scss/variables/_colors.scss

```diff
@@ -0,0 +1,45 @@
+$c-white: #ffffff;
+$c-red: #ea5555;
+$c-purple: #635fc7;
+
+// grey tones
+$c-grey-100: #efeffa;
+$c-grey-200: #f4f7fd;
+$c-grey-300: #828fa3;
+$c-grey-400: #2b2c37;
+$c-grey-500: #20212c;
+$c-grey-600: #000112;
+
+// light
```

Comment:
This is already in theme.scss. Please, remove it

Reply:
done